.github/workflows/vulnerability-scanning-on-repo.yml

```diff
@@ -11,15 +11,19 @@ on:
         required: false
         default: .
         type: string
+      package-directory:
```

Comment:
Similar comment to Daniel, at some point we need to make a stylistic call about upper case vs lower case inputs, typically our convention to date has been upper case